### PR TITLE
PCC03

### PR DESCRIPTION
**Pull request by code-monk08**
ATTENTION: before clicking "Create Pull Request" please submit some meta data, thanks!

Difficulty level (1-10): 7
Estimated time spent (hours): 0.45
Completed (yes/no): yes
I stretched my coding skills (if yes what did you learn?): learning more about collections and itertools modules was fun
Other feedback (what can we improve?): all ok

### CODE REVIEW

File: 03/code-monk08/tags.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+def get_top_tags(tags):
+    return Counter(tags).most_common(TOP_NUMBER)
+
+
+def get_similarities(tags):
+    result = {}
+    for x, y in product(tags, tags):
+        if x == y or x[0] != y[0]:
+            continue
+        else:
```
Comment (pybites) [inline 03/code-monk08/tags.py]: when you `continue` you probably don't need the `else`

File: 03/code-monk08/tags.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+SIMILAR = 0.87
+TAG_HTML = re.compile(r'<category>([^<]+)</category>')
+
+
+def get_tags():
+    return TAG_HTML.findall(open(RSS_FEED).read())
+
+
+def get_top_tags(tags):
+    return Counter(tags).most_common(TOP_NUMBER)
```
Comment (pybites) [inline 03/code-monk08/tags.py]: love `Counter`

**Comment by pybites** [comment]
Good job

**Comment by pybites** [comment]
Remember to PR against the community branch, if you use the link presented on the platform, it should choose that branch automatically.

File: 03/code-monk08/tags.py (hunk 1 above)
Comment (code-monk08) [reply 03/code-monk08/tags.py]: Thanks, will take a note of that.

**Pull request merged**